PrebidMobileTests/RenderingTests/Tests/PBMORTBAbstractTest.swift

```diff
@@ -174,6 +174,11 @@ class PBMORTBAbstractTest : XCTestCase {
         pbmORTBBidRequest.test = 2
         
         codeAndDecode(abstract: pbmORTBBidRequest, expectedString: "{\"id\":\"\(uuid)\",\"imp\":[{\"clickbrowser\":0,\"ext\":{\"dlp\":1},\"instl\":0,\"secure\":0}],\"test\":2,\"tmax\":2000}")
+        
+        pbmORTBBidRequest.ortbObject = ["arbitraryparamkey1": "arbitraryparamvalue1", "tmax": 3000, "id": "1231234", "imp": ["hi": "hello"]]
+        
+        codeAndDecode(abstract: pbmORTBBidRequest, expectedString: "{\"arbitraryparamkey1\":\"arbitraryparamvalue1\",\"id\":\"\(uuid)\",\"imp\":[{\"clickbrowser\":0,\"ext\":{\"dlp\":1},\"instl\":0,\"secure\":0}],\"test\":2,\"tmax\":2000}")
```

Comment:
We need to cover more use cases here, including incorrect JSONs. Not overriding of reserved objects. Also the custom parameter should be a String but not Dictionary.